bpf/lib/policy_log.h

```diff
@@ -57,8 +57,13 @@ send_policy_verdict_notify(struct __ctx_buff *ctx, __u32 remote_label, __u16 dst
 	__u64 cap_len = min_t(__u64, TRACE_PAYLOAD_LEN, ctx_len);
 	struct policy_verdict_notify msg;
 
+#if defined(IS_BPF_HOST)
```

Comment:
> BTW, why can't we fix POLICY_VERDICT_LOG_FILTER for host firewall instead of making an exception?

it would be a bit involved to reload bpf_host with the appropriate POLICY_VERDICT_LOG_FILTER whenever there's at least one host firewall policy, and also given that it's bpf_host we would still need to solve the issue with the flood of `allow all` events, so we decided to go for this simpler way